antarest/study/storage/rawstudy/io/reader/ini_reader.py

```diff
@@ -77,6 +77,53 @@ def read(self, path: Path) -> JSON:
         }
 
 
+class XpansionSettingsIniReader(IReader):
```

Comment:
You can rename it SimpleKeyValueReader since we may want to reuse it for other files